meta.js

```diff
@@ -105,6 +105,7 @@ module.exports = {
     ".eslintrc.js": "lint",
     ".eslintignore": "lint",
     "config/test.env.js": "unit || e2e",
+    "build/webpack.test.conf.js": "unit || e2e",
```

Comment:
I think that config is only required if `e2e || (unit && runner === 'karma')` - jest doesn't use webpack.